Dockerfiles/dogstatsd/alpine/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG BASE_IMAGE_ALPINE_VERSION=3.18
-FROM alpine:$BASE_IMAGE_ALPINE_VERSION
+FROM public.ecr.aws/docker/library/alpine:$BASE_IMAGE_ALPINE_VERSION
```

Comment:
`alpine` images should be mirrored on our internal ECR: https://github.com/DataDog/images/blob/f8fe2c7f82f5475b240858625361a33c32da1892/mirror.yaml#L439-L483
Using our mirror would remove the dependency to a public repo.